spacy/cli/project/remote_storage.py

```diff
@@ -119,16 +118,19 @@ def find(
         name = self.encode_name(str(path))
         if command_hash is not None and content_hash is not None:
             url = self.make_url(path, command_hash, content_hash)
-            urls = [url] if url.exists() else []
+            urls = [url] if _file_exists(url) else []
         elif command_hash is not None:
             urls = list((self.url / name / command_hash).iterdir())
         else:
-            urls = list((self.url / name).iterdir())
+            urls = []
+            for command_hash_dir in (self.url / name).iterdir():
+                urls.extend(command_hash_dir.iterdir())
             if content_hash is not None:
                 urls = [url for url in urls if url.parts[-1] == content_hash]
+        # TODO: URLs should be sorted by last modified
```

Comment:
`pathy` doesn't return file stats in the same format for local paths and remote paths, so this would still need an additional helper method to sort the files by last modified.

Reply:
Can you elaborate on what the problem is? I'd be happy to look at making it more consistent across local/remote paths.

Reply:
It looks like you get `BlobStat` for remote paths and `os.stat_result` for local paths?

Reply:
You're right that calling `.stat()` on a local file system path was returning `os.stat_result`. 

I released Pathy `0.10.0` today which returns `BlobStat` for local and remote paths.

Reply:
Thanks, we'll try it out!